docs/ref/models/class.txt

```diff
@@ -11,6 +11,35 @@ reference guides </ref/models/index>`.
 Attributes
 ==========
 
+``DoesNotExist``
+----------------
+
+.. exception:: Model.DoesNotExist
+
+    This exception is raised by the ORM in a couple places, for example by
+    :meth:`QuerySet.get() <django.db.models.query.QuerySet.get>` when an object
+    is not found for the given lookups.
+
+    Django provides a ``DoesNotExist`` exception as an attribute of each model
+    class to identify the class of object that could not be found and to allow
+    you to catch a particular model class with ``try/except``. The exception is
+    a subclass of :exc:`django.core.exceptions.ObjectDoesNotExist`.
+
+``MultipleObjectsReturned``
+---------------------------
+
+.. exception:: Model.MultipleObjectsReturned
+
+    This exception is raised by
+    :meth:`QuerySet.get() <django.db.models.query.QuerySet.get>` when multiple
+    objects are found for the given lookups.
+
+    Django provides a ``MultipleObjectsReturned`` exception as an attribute of
+    each model class to identify the class of object that for which multiple
+    objects were found and to allow you to catch a particular model class with
+    ``try/except``. The exception is a subclass of
```

Comment:
`that for which` looks like an error in making this similar to the text for `DoesNotExist` above.

`…class of object for which multiple objects were found allowing capture of the exception for that specific class with ``try/except``.`?